cmd/influxd/run/config.go

```diff
@@ -39,11 +40,12 @@ const (
 
 // Config represents the configuration format for the influxd binary.
 type Config struct {
-	Meta        *meta.Config       `toml:"meta"`
-	Data        tsdb.Config        `toml:"data"`
-	Coordinator coordinator.Config `toml:"coordinator"`
-	Retention   retention.Config   `toml:"retention"`
-	Precreator  precreator.Config  `toml:"shard-precreation"`
+	Meta           *meta.Config       `toml:"meta"`
+	Data           tsdb.Config        `toml:"data"`
+	Coordinator    coordinator.Config `toml:"coordinator"`
+	FluxController control.Config     `toml:"flux-controller"`
```

Comment:
I like having a separate config section for Flux, but will it be confusing for users that the toggle to enable flux is under the `HTTPD` config instead of this new section? Should we consider creating an "enabled" toggle under here and deprecating the old one?

Reply:
I see this as the query controller is supposed to be global, but the http `enabled` toggle just tells you if you're allowed to access flux via http. In theory we could have other API's that access the same flux controller.